frontend/src/components/UserProfile.tsx

```diff
@@ -1,45 +1,52 @@
-import {
-  Box,
-  Button,
-  Flex,
-  ListItem,
-  Text,
-  UnorderedList,
-} from "@chakra-ui/react";
+import { Avatar, Button, Center, Flex, Text } from "@chakra-ui/react";
+import { useAsync } from "react-async-hook";
 import { Link } from "react-router-dom";
+import { getUserDetails } from "../services/songs";
 
 export default function UserProfile() {
+  const asyncUser = useAsync(async () => getUserDetails(), []);
```

Comment:
This isn't important, but thought you might want to know: useAsync expects an async function as it's first param, which means you can just pass it `getUserDetails` for that.

Reply:
My apologies. I copied from the useAsync example in SongbookList, but I failed to note that the async callback in that instance requires an argument.